tests/subsys/usb/negotiated_speed/testcase.yaml

```
@@ -0,0 +1,15 @@
+common:
+  tags:
+    - usb
+    - ci_tests_subsys_usb
+  harness: pytest
+
+tests:
+  test.usb.negotiated_speed:
```

Please change name of the test:
```suggestion
  usb.negotiated_speed:
```